app/src/main/res/values/dimens.xml

```diff
@@ -22,4 +22,5 @@
     <dimen name="onboarding_dots_corner_radius">1.5dp</dimen>
     <dimen name="onboarding_dots_size">4dp</dimen>
     <dimen name="onboarding_dots_spacing">8dp</dimen>
+    <dimen name="button_height">56dp</dimen>
```

Comment:
Remember to look over your res files after you added some values so you arrange them properly. Unfortunately AS doesn't recognize automatically where they should be even with region tags :(